README.md

```diff
@@ -66,10 +200,117 @@ The module automatically adds an "SEO" tab to all page and piece editors contain
 - **Meta Description**: Compelling descriptions that appear in search results
 - **Robots Meta Tag**: Control search engine indexing and following behavior
 - **Canonical URLs**: Prevent duplicate content penalties
+- **Schema Type Selection**: Choose the appropriate structured data type for your content
+
+### Google Analytics & Tag Manager
+
+Built-in integration with Google Analytics, Google Tag Manager, and Google Site Verification. Simply enable the options you need and add your tracking IDs through the global configuration interface.
+
+**Supported integrations:**
+- Google Analytics (GA4) tracking
+- Google Tag Manager for advanced marketing campaigns
+- Google Site Verification for Search Console
+
+See [Essential Configuration](#essential-configuration) below for setup instructions.
+
+### Automated Robots.txt
+
+The module automatically provides a `/robots.txt` route with strategic control over both traditional search engines and AI crawlers. Configure through global settings with five control modes:
+
+**Available Modes:**
+
+1. **Allow All (Search + AI)** - Default open access for all crawlers
+2. **Allow Search, Block AI Training** - Maintains search rankings while protecting content from AI training
+3. **Selective AI Crawlers** - Granular control over individual AI crawlers
+4. **Block All** - Prevents all indexing
+5. **Custom** - Write your own robots.txt content
+
+**Selective Mode Crawlers:**
+For fine-grained control, use Selective mode to choose specific AI crawlers:
```

Comment:
Interesting to see so many that honor this and/or at least use a user agent. I know for a fact our clients get a crapton of spider traffic now that masquerades as a browser. (Just an observation not a request for changes)